neofetch

```diff
@@ -3202,6 +3202,49 @@ END
             # Default fallback font hardcoded in terminal-preferences.c
             [[ -z "$term_font" ]] && term_font="Monospace 12"
         ;;
+
+        "conemu-"*)
+            local ce_arg_list
+            local ce_arg_idx
+            local ce_conf
+
+            # Could have used `eval set -- "$ConEmuArgs"` instead for arg parsing but eval is evil...
+            readarray -t ce_arg_list < <(xargs -n1 printf "%s\n" <<< "${ConEmuArgs-}")
+
+            for ce_arg_idx in "${!ce_arg_list[@]}"; do
+                # Search for "-LoadCfgFile" arg
+                [[ "${ce_arg_list[$ce_arg_idx]}" != -LoadCfgFile ]] && continue
+
+                # Conf path is the next arg
+                ((++ce_arg_idx))
+                ce_conf="${ce_arg_list[$ce_arg_idx]}"
+                break
+            done
+
+            # https://conemu.github.io/en/ConEmuXml.html#search-sequence
+            local ce_seq=(
+                "$ce_conf"
+                "$ConEmuDir\ConEmu.xml"
+                "$ConEmuDir\.ConEmu.xml"
+                "$ConEmuBaseDir\ConEmu.xml"
+                "$ConEmuBaseDir\.ConEmu.xml"
+                "$APPDATA\ConEmu.xml"
+                "$APPDATA\.ConEmu.xml"
+            )
+
+            for ce_conf in "${ce_seq[@]}"; do
+                # Search for first conf file available
+                [[ ! -f "$ce_conf" ]] && continue
+
+                # Very basic XML parsing
+                term_font="$(awk '/name="FontName"/ && match($0, /data="([^"]*)"/) {print substr($0, RSTART+6, RLENGTH-7)}' "$ce_conf")"
+                break
+            done
+
+            # Null-terminated contents in /proc/registry files triggers a Bash warning. Use read instead
+            [[ -z "$term_font" ]] && \
```

Comment:
```sh
[[ "$term_font" ]] || \
```

(You may be able to squeeze this onto a single line with the above change)

Reply:
I just found there is a line length check in `.travis.yml`. I'll apply this change but the `read -r term_font` is still too long. I'll try a different wrapping.